src/platform/graphics/graphics-device.js

```diff
@@ -30,12 +30,11 @@ class GraphicsDevice extends EventHandler {
     canvas;
 
     /**
-     * The graphics device type, DEVICETYPE_WEBGL or DEVICETYPE_WEBGPU.
+     * True if the deviceType is WebGPU
      *
-     * @type {string}
-     * @ignore
+     * @type {boolean}
      */
-    deviceType;
+    isWebGPU = false;
```

Comment:
Isn't this property's naming a bit different to `WebglGraphicsDevice#webgl2`?

Reply:
it is, and I considered renaming webgl2 to isWebGl2. What do you think about this?

Reply:
Maybe. I think we're starting to mix different styles of capitalization for WebGL/GPU. There's `Webgl`, WebGl`, `WebGL`.